content/GettingStarted/VersionNotes.md

```diff
@@ -291,20 +336,27 @@ These are ongoing issues that can affect multiple versions in the 25.10 series.
 
 ### Current Known Issues
 
-* Samba 4.22 has a bug that breaks time machine backups for some newer MacOS versions ([NAS-138644](https://ixsystems.atlassian.net/browse/NAS-138644)).
-  TrueNAS 25.04 releases are unaffected. This is fixed in TrueNAS 25.10.1 with the inclusion of Samba update 4.22.6.
-
-* Hot spare vdevs are limited to a single disk per pool in the UI ([NAS-138640](https://ixsystems.atlassian.net/browse/NAS-138640)).
-  The TrueNAS 25.10 UI currently restricts configuration of hot spare vdevs to one disk per pool.
-  A fix for this issue is expected in an upcoming maintenance release.
-
 * Apps using SMB/NFS storage can experience race condition during boot.
   When apps are configured to use SMB or NFS shares as storage passthroughs, there can be an occasional race condition during TrueNAS boot where the app startup conflicts with the sharing services startup.
   This causes affected apps to not fully start and show a "crashed" status.
 
   Workaround: Restart the affected app after a minute or two.
-  
-  Additional fixes to this area are expected in the 25.10.1 maintenance release.
+
+* Long-running systems with legacy FreeNAS datasets may encounter dataset editing errors ([NAS-138629](https://ixsystems.atlassian.net/browse/NAS-138629)).
+  Users may see `"aclmode: failed to get property"` errors when attempting to edit dataset properties. This affects datasets that contain invalid `aclmode=discard_chmod` property values from legacy FreeNAS versions.
+
+  Workaround: Run `zfs set aclmode=passthrough dataset_name` via CLI (replacing `dataset_name` with the actual dataset path), then reboot the system.
+
+  Error messaging improvements for this issue are included in TrueNAS 25.10.1 to help identify affected datasets.
+
+* Removing RBAC roles from the root account can cause scheduled tasks to fail.
+  The 25.10 UI allows modification of the root account's role assignments. Removing the **FULL_ADMIN** role from the root account can cause Cloud Sync tasks, cron jobs, and other scheduled operations to fail.
+
+  To disable root account access to the TrueNAS UI, use the **Disable Password** option in Credentials > Local Users instead of removing RBAC roles.
+
+  Workaround: If scheduled tasks are not running after modifying root account roles, re-add the **FULL_ADMIN** role to the root account in Credentials > Local Users. If needed, use **Disable Password** to prevent root UI access while maintaining proper system operation.
+
+  Future TrueNAS releases will include additional validation to prevent removal of required root account privileges.
```

Comment:
"will"
```suggestion
  Future TrueNAS releases are planned to include additional validation to prevent removal of required root account privileges.
```